docs/CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+  * **pois**: Consolidate `graveyard` and `cemetery` to share same zoom ranges (Issue [#1780](https://github.com/tilezen/vector-datasource/issues/1780))
+  * **pois**: Fix junk `healthcare` values and kinds introduced in v1.5. Added: `chiropractor`, `hospice`, `occupational_therapist`, `optometrist`, `paediatrics`, `physiotherapist`, `podiatrist`, `psychotherapist`, `rehabilitation`, and `speech_therapist` kinds. (Issue [#1596](https://github.com/tilezen/vector-datasource/issues/1596))
+  * **pois**: Use same tier 2 zoom range for `nature_reserve` POIs as the landuse polygons.
+  * **pois**: Fix tier 3 (and 1 & 4) to not duplicate zoom 12 threshold.
+  * **pois**: Fix tier3 POIs zoom 12 threshold to 200000 area (Issue [#1769](https://github.com/tilezen/vector-datasource/issues/1769))
+  * **pois**: Simplify most tier 1 & 2 POI min zooms to area-based thresholds.
+  * **pois**: Default `fitness_station` to zoom 18 instead of 17.
+  * **pois**: Whitelist `kind_detail` for `religion` related kinds to just: `animist`, `bahai`, `buddhist`, `caodaism`, `catholic`, `christian`, `confucian`, `hindu`, `jain`, `jewish`, `multifaith`, `muslim`, `pagan`, `pastafarian`, `scientologist`, `shinto`, `sikh`, `spiritualist`, `taoist`, `tenrikyo`, `unitarian_universalist`, `voodoo`, `yazidi`, and `zoroastrian`.
+  * **pois**: Whitelist `kind_detail` for `health_facility` kind to just types: `CSCom`, `chemist_dispensing`, `clinic`, `counselling_centre`, `dispensary`, `first_aid`, `health_center`, `health_centre`, `hospital`, `laboratory`, `medical_clinic`, `office`, `pharmacy`.
+  * **pois**: Whitelist `kind_detail` for `sports` related kinds to just: `10pin`, `9pin`, `american_football`, `archery`, `athletics`, `badminton`, `baseball`, `basketball`, `beachvolleyball`, `billiards`, `bmx`, `boules`, `bowls`, `canoe`, `chess`, `climbing`, `cricket`, `cricket_nets`, `cycling`, `equestrian`, `exercise`, `field_hockey`, `fitness`, `football`, `free_flying`, `futsal`, `gaelic_games`, `golf`, `gymnastics`, `handball`, `hockey`, `horse_racing`, `ice_hockey`, `ice_skating`, `karting`, `model_aerodrome`, `motocross`, `motor`, `multi`, `netball`, `padel`, `pelota`, `rugby`, `rugby_league`, `rugby_union`, `running`, `scuba_diving`, `shooting`, `skateboard`, `skating`, `skiing`, `soccer`, `soccer;basketball`, `softball`, `swimming`, `table_tennis`, `team_handball`, `tennis`, `trampoline`, `volleyball`, `yoga`.
+  * **pois**: Whitelist `kind_detail` for `cousine` related kinds to just: `american`, `asian`, `barbecue`, `breakfast`, `burger`, `cake`, `chicken`, `chinese`, `coffee_shop`, `crepe`, `donut`, `fish`, `fish_and_chips`, `french`, `friture`, `georgian`, `german`, `greek`, `ice_cream`, `indian`, `international`, `italian`, `japanese`, `kebab`, `korean`, `lebanese`, `local`, `mediterranean`, `mexican`, `noodle`, `pizza`, `ramen`, `regional`, `sandwich`, `seafood`, `spanish`, `steak_house`, `sushi`, `tapas`, `thai`, `turkish`, `vegetarian`, `vietnamese`.
```

Comment:
Spelling `cousine` -> `cuisine`

Reply:
Well caught! Fixed in a76eda8099e66fb27aaeea38709ef58b4ec3d6ef.